### ISSUE TITLE
.wbl file missing

### ISSUE THREAD

**Issue by simonaubertbd** [leematthewshome/Qlik-Sense-Live-Table#5]
Hello Lee,

Is it possible to have the .wbl file with the extension?

I know how to write myself (just the list of file with a ; and a carriage return) however I think it would be fine to have it with the other files ;)

Best regards,

Simon

**Comment by leematthewshome** [comment]
I dont have the wbl file on my PC ans it seems to work fine without it on desktop. Does the absence of this cause a problem with the server version?

**Comment by FocusSam** [comment]
> I dont have the wbl file on my PC ans it seems to work fine without it on desktop. Does the absence of this cause a problem with the server version?

Yes, I can not open and use your extension on Server version. There is an error "The LiveTable is incomplete and cannot be opened or duplicated (wbfolder.wbl file missing).
